### DISCUSSION TITLE
Colors used for Air Quality input field on Mytourbook 23-5

### DISCUSSION THREAD

**Discussion by witdem** [mytourbook/mytourbook#1108] [General]
The upcoming version 23.5 has an input field for air quality. 
The values however do have a colored background depending on the values (see attached screenshot) 

It is nice to have the air quality, but I think that this dropdown field should have a non-colored background like all other input/dropdowns. In graphs colors are an added value, but here it distracts me 

What is the opinion of others?     

![afbeelding](https://github.com/mytourbook/mytourbook/assets/14309258/37807801-7a16-48b9-951d-ba0351747b38)

**Comment by FJBDev** [comment]
It will be interesting to see what others think but here is why I had the idea of using the colored background:

When I learned about the air quality, I found that most of the used [air quality scales](https://en.wikipedia.org/wiki/Air_quality_index) use background and found that it makes it easier than a white background and help "the eye" to quickly identify if it's good or bad air.

![image](https://github.com/mytourbook/mytourbook/assets/25235665/7e7bb344-e047-4ea5-9d0e-e34cbffbc0c5)

However, if people prefer white background, I am ok reverting to a "normal" dropdown, especially since the Nebula widgets are "half-baked" and have issues sometimes

**Reply by wolfgang-ch** [reply]
> ...since the Nebula widgets are "half-baked" and have issues sometimes

An alternative could be [NatCombo](https://download.eclipse.org/nattable/releases/1.4.0/apidocs/org/eclipse/nebula/widgets/nattable/widget/NatCombo.html).

[NatTable](https://www.eclipse.org/nattable/) is used in the flat tourbook view and is very reliabled but also highly complicated however NatCombo may be less complicated.

**Comment by wolfgang-ch** [comment]
When the colors could be customized, then both will be happy 😄 

I just improved the colors for the categorized views to support both themes, dark and bright

![image](https://github.com/mytourbook/mytourbook/assets/1283445/577e3599-01a7-41c1-a765-d60362f2be61)

**Comment by FJBDev** [comment]
Maybe we could also wait that 23.5 is released so that more users can try it and have an opinion and then when they share their opinion we can either keep the Nebula widget or use a standard SWT widget ?

**Comment by wolfgang-ch** [comment]
For 23.5 we will not modify it but afterwards we can

**Comment by stvzldn** [comment]
AQI in general does not include pollen indices.  Is there any consideration for logging pollen as well as AQI?
This site gives values for both AQI and Pollen https://www.breezometer.com/air-quality-map/air-quality/united-states/chicago

**Reply by FJBDev** [reply]
> AQI in general does not include pollen indices. Is there any consideration for logging pollen as well as AQI? This site gives values for both AQI and Pollen https://www.breezometer.com/air-quality-map/air-quality/united-states/chicago

The c[urrent API used](https://openweathermap.org/api/air-pollution) doesn't include pollen data

**Reply by stvzldn** [reply]
I inquired at openweathermap if they plan to include pollen index value in the API.  Below is the response:

> At the moment, we do not return such data. However, in our long-term plans, we have the task of thinking about this parameter and product. I will send your wishes and your feedback to the development department.
